CovertInfiltration/Config/XComGameBoard.ini

```diff
@@ -252,6 +284,28 @@ MaxActionHours[1]=192
 MaxActionHours[2]=192
 MaxActionHours[3]=240
 
+[CovertAction_PatrolWilderness X2CovertActionTemplate]
+MinActionHours[0]=120
+MinActionHours[1]=120
+MinActionHours[2]=120
+MinActionHours[3]=168
+
+MaxActionHours[0]=168
+MaxActionHours[1]=168
+MaxActionHours[2]=168
+MaxActionHours[3]=240
```

Comment:
Shouldn't this be short to account for time you will spend scanning the POIs anyway?